data/bilibili

```diff
@@ -0,0 +1,8 @@
+bilibili.com
+www.im9.com
```

Comment:
this site is no longer active.

Reply:
But these domains was provided by official application
![screenshot](https://q1q2q3.tk/wp-content/uploads/2018/12/IMG_20181213_000146.jpg)

Reply:
Ok. Please remove `www.` at beginning.

Reply:
Like this?